### ISSUE TITLE
Support Python >= 3.10

### ISSUE THREAD

**Issue by sschuberth** [bpozdena/OneDriveGUI#130]
Fedora 38 (KDE spin) comes with Python 3.11 by now, but which that it fails to build:

```
$ python3 -m pip install -r requirements.txt
Defaulting to user installation because normal site-packages is not writeable
ERROR: Ignored the following versions that require a different python version: 6.0.0 Requires-Python >=3.6, <3.10; 6.0.0a1.dev1606911628 Requires-Python >=3.6, <3.10; 6.0.1 Requires-Python >=3.6, <3.10; 6.0.2 Requires-Python >=3.6, <3.10; 6.0.3 Requires-Python >=3.6, <3.10; 6.0.4 Requires-Python >=3.6, <3.10; 6.1.0 Requires-Python >=3.6, <3.10; 6.1.1 Requires-Python >=3.6, <3.10; 6.1.2 Requires-Python >=3.6, <3.10; 6.1.3 Requires-Python >=3.6, <3.10; 6.2.0 Requires-Python >=3.6, <3.11; 6.2.1 Requires-Python >=3.6, <3.11; 6.2.2 Requires-Python >=3.6, <3.11; 6.2.2.1 Requires-Python >=3.6, <3.11; 6.2.3 Requires-Python >=3.6, <3.11; 6.2.4 Requires-Python >=3.6, <3.11; 6.3.0 Requires-Python <3.11,>=3.6; 6.3.1 Requires-Python <3.11,>=3.6; 6.3.2 Requires-Python <3.11,>=3.6; 6.4.0 Requires-Python <3.11,>=3.6
ERROR: Could not find a version that satisfies the requirement PySide6==6.2.2.1 (from versions: 6.4.0.1, 6.4.1, 6.4.2, 6.4.3, 6.5.0, 6.5.1, 6.5.1.1, 6.5.2, 6.5.3, 6.6.0)
ERROR: No matching distribution found for PySide6==6.2.2.1
```

**Comment by bpozdena** [comment]
I've updated the requirements, but for now only in the [alpha branch](https://github.com/bpozdena/OneDriveGUI/blob/v1.1.0-alpha0/requirements.txt).

**Comment by sschuberth** [comment]
Thanks! By now Fedora released version 39, coming with Python 3.12 by default. Is that supported already as well in the `alpha` branch?

**Comment by bpozdena** [comment]
@sschuberth , I would not expect any issues on Python 3.12 even on the main branch. It should work with any current version of PySide6 and requests libraries as well.
